src/hearings/containers/request-hearing/request-hearing.component.html

```diff
@@ -2,9 +2,9 @@
 <div class="govuk-width-container">
   <div class="page">
     <main id="content" role="main">
-      <p>
-        request-hearing works!
-      </p>
+        <div class="search-location">
+          <exui-search-location class="search-location"></exui-search-location>
+        </div>     
```

Comment:
@Kartheek-S has a pr https://github.com/hmcts/rpx-xui-webapp/pull/1481 which should be merged first, so your pr should depends on his, and this changes is not necessary anymore if we use parent route. As this seems duplicated with location-search-container.component.html